package.json

```diff
@@ -4,7 +4,12 @@
   "description": "NodeJS Global Mentoring Program Hometasks",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "task-1.1": "nodemon module-1/task-1",
+    "task-1.2": "nodemon module-1/task-2",
+    "build-task-1.1": "babel module-1/task-3/task-1 -d module-1/task-3/task-1/dist",
```

Comment:
Typically, when just a Vanilla JS is used, there's no need to create a build on the server.
The server process is launched and it handles the transformations (such as babel) on the fly.
Therefore, you could've used babel-node and instruct nodemon to use it on the fly using "--exec".
https://www.codementor.io/@michaelumanah/how-to-set-up-babel-7-and-nodemon-with-node-js-pbj7cietc

Reply:
Fixed, thanks!

Reply:
Fixed, thanks!